### ISSUE TITLE
Support for dynamic imports in body

### ISSUE THREAD

**Issue by corvana-nbaker** [standard/standard#1122]
We're leveraging dynamic imports and standard keeps complaining: "Parsing error: 'import' and 'export' may only appear at the top level"

Eslint supports suppressing this via the "allowImportExportEverywhere" parser option, but standard doesn't seem to read the .eslintrc file. /* eslint-disable /* doesn't work either.

Is anyone aware of a way to ignore this check?

**Comment by LinusU** [comment]
`/* eslint-disable */` should work...

May I ask how your setup is? I thought that dynamic imports where made via a function call (e.g. `import('./foobar').then(() => /* ... */)`) instead of using the import keyword?

**Comment by feross** [comment]
@corvana-nbaker Are you referring to the `allowImportExportEverywhere` option that `babel-eslint` takes?

**Comment by corvana-nbaker** [comment]
Yes! How can I get "allowImportExportEverywhere" to take effect with standard?

**Comment by feross** [comment]
It appears that the `allowImportExportEverywhere` option is specifically for allowing `import` and `export` **statements** to work anywhere, effectively emulating how `require()` works.

I don't think this should be necessary if you're using the dynamic `import()` syntax.

Using `babel-eslint@8` and `standard@11`, I am unable to reproduce this issue. It lints `import()` perfectly fine as far as I can tell. 

Also, this appears related to: https://github.com/standard/standard/issues/1041
